code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -173,22 +173,22 @@
 		else
 			if(decal.random_icon_states && length(decal.random_icon_states) > 0) //If there's already one, just randomize it so it changes.
 				decal.icon_state = pick(decal.random_icon_states)
-
+		
+		if(!(xeno_caste.caste_flags & CASTE_ACID_BLOOD))
+			return
 		var/splash_chance = 40 //Base chance of getting splashed. Decreases with # of victims.
 		var/distance = 0 //Distance, decreases splash chance.
 		var/i = 0 //Tally up our victims.
+			for(var/mob/living/carbon/human/victim in range(radius,src)) //Loop through all nearby victims, including the tile.
```

Comment:
you have a space at the start of this line per this: https://github.com/SpaceManiac/SpacemanDMM/issues/117

Reply:
No wait, you indented too much